gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -979,6 +981,27 @@ function GM:OnContextMenuOpen()
 	end
 end
 
+-- Preload materials for the shop
+hook.Add("PostInitPostEntity", "TTTCacheEquipMaterials", function()
```

Comment:
I guess we should name new hook functions consistently: so i'd suggest `TTT2CacheEquipmentMaterials`